polaris.shopify.com/pages/[...slug].tsx

```diff
@@ -94,6 +94,7 @@ export const getStaticPaths: GetStaticPaths = async () => {
     path.resolve(process.cwd(), 'content/*.md'),
     path.resolve(process.cwd(), 'content/**/*.md'),
     path.resolve(process.cwd(), 'content/**/**/*.md'),
+    path.resolve(process.cwd(), 'content/**/**/**/*.md'),
```

Comment:
Only need this glob path for this file, not [here](https://github.com/Shopify/polaris/pull/8035/files#r1069655466)